.github/workflows/data_sync.yml

```diff
@@ -8,6 +8,10 @@ on:
   schedule:
     - cron: '0 0 * * MON'
   workflow_dispatch:
+    version:
+      description: 'Would you like to specify a version for dob_jobapplications?'
```

Comment:
Is the reason we're not specifying dob_now versions because the updates are manual and infrequent?

Reply:
yes